gcloud/datastore/connection.py

```diff
@@ -317,9 +304,9 @@ def commit(self, project, request_pb):
         request_pb.project_id = project
         try:
             return self._stub.Commit(request_pb, GRPC_TIMEOUT_SECONDS)
```

Comment:
Timeouts are now optional. So "always pass positional arguments positionally and always pass keyword arguments by keyword", or just kill the argument if you never wanted to pass it in the first place.